server/models/Board.js

```diff
@@ -15,12 +15,17 @@ const BoardSchema = new Schema(
         ref: "Column",
       },
     ],
+    creator: {
+      type: Schema.Types.ObjectId,
+      required: true,
+    },
+    members: [],
```

Comment:
I think the "members" property should also have
`type: Schema.Types.ObjectId`

Reply:
Emails will be stored in the members array, so the "type: Schema.Types.ObjectId" is not required. I could add the "type: String" inside though.